functions/src/services/message/defaultMessageService.ts

```diff
@@ -338,16 +338,13 @@ export class DefaultMessageService implements MessageService {
 
     switch (input.message.content.type) {
       case UserMessageType.weightGain:
-        if (!user.receivesRecommendationUpdates) return
         break
       case UserMessageType.welcome:
         break
       case UserMessageType.vitals:
       case UserMessageType.symptomQuestionnaire:
```

Comment:
Maybe also remove the message types? You don't really make use of most of them anyways, right?